modules/hostap/src/supp_api.c

```diff
@@ -1173,6 +1183,73 @@ int supplicant_11k_neighbor_request(const struct device *dev, struct wifi_11k_pa
 	return 0;
 }
 
+#ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_ROAMING
+#define SUPPLICANT_CANDIDATE_SCAN_CMD_BUF_SIZE 100
+int supplicant_candidate_scan(const struct device *dev, struct wifi_scan_params *params)
+{
+	int i = 0;
+	char cmd[SUPPLICANT_CANDIDATE_SCAN_CMD_BUF_SIZE] = {0};
+	char *pos = cmd;
+	char *end = pos + SUPPLICANT_CANDIDATE_SCAN_CMD_BUF_SIZE;
+	int freq = 0;
+
+	strcpy(pos, "freq=");
+	STR_CUR_TO_END(pos);
```

Comment:
Here you could just say `pos += 5;` as we always copy the same string. There really is no need to do `strlen()`

Reply:
Updated.